charts/airbyte-server/templates/_helpers.tpl

```diff
@@ -63,10 +63,10 @@ Define imageTag
 */}}
 
 {{- define "server.imageTag" -}}
-{{- if ((.Values.global.image).tag) }}
-    {{- printf "%s" .Values.global.image.tag }}
 {{- else if .Values.image.tag }}
```

Comment:
```suggestion
{{- if .Values.image.tag }}
```

Reply:
It doesn't work as expected, sample output:
`image: airbyte/bootloader:test2test`

Reply:
Values.yaml reference:
```
##  global.serviceAccountName Name of service account to be associated with service
##  deploymentMode Determines deployment mode of airbyte
##  secretName Overrides the secrate name with with credentials to S3 Bucket and logging
##  database.secretName Name of database secret
##  database.secretValue Value of database password key stored in secret
##  database.host Hostname of database
##  database.port Database port
global:
  serviceAccountName: placeholderServiceAccount
  deploymentMode: oss
  secretName: ""
  database:
    secretName: ""
    secretValue: ""
    host: "example.com"
    port: "5432"
  secrets: {}
  env_vars: {}
  extraContainers: []
  image:
    tag: test


enabled: true
##  image.repository The repository to use for the airbyte bootloader image.
##  image.pullPolicy the pull policy to use for the airbyte bootloader image
##  image.tag The airbyte bootloader image tag. Defaults to the chart's AppVersion
image:
  repository: airbyte/bootloader
  pullPolicy: IfNotPresent
  tag: test2
```

Reply:
Sorry, my mistake, fixed the typo, pushed changes to this branch.